dotnet/src/SemanticKernel.Abstractions/Kernel.cs

```diff
@@ -212,6 +222,28 @@ internal bool OnFunctionInvoked(FunctionInvokedEventArgs eventArgs)
         }
         return handled;
     }
+
+    internal bool OnPromptRendering(PromptRenderingEventArgs eventArgs)
+    {
+        bool handled = false;
+        if (this.PromptRendering != null)
```

Comment:
nit, apply to other places.
```suggestion
        if (this.PromptRendering is not null)
```